### PR TITLE
feat(route): 新增知闻AI资讯

### PR DESCRIPTION
**Pull request by guicaiyue**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例
https://informedainews.com/zh-Hans/docs/ai-enthusiast-daily

```routes
/informedainews/zh-Hans/docs/world-news-daily
/informedainews/zh-Hans/docs/tech-enthusiast-weekly
/informedainews/zh-Hans/docs/ai-enthusiast-daily
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/informedainews/docs.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/zh-Hans/docs/:type',
+    categories: ['journal'],
+    example: '/docs/world-news-daily',
```
Comment (TonyRL) [inline lib/routes/informedainews/docs.ts:10]:
```suggestion
    example: '/informedainews/zh-Hans/docs/world-news-daily',
```

File: lib/routes/informedainews/docs.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            const start = text.indexOf('(');
+            // 找到第一个')'字符的位置
+            const end = text.indexOf(')');
+            // 从第一个'('到第一个')'之间的子字符串就是日期
+            const date = text.substring(start + 1, end);
+            return {
+                title: text,
+                link: `https://informedainews.com${a.attr('href')}`,
+                pubDate: parseDate(date),
+                author: `AI`,
```
Comment (TonyRL) [inline lib/routes/informedainews/docs.ts:51]:
```suggestion
                author: 'AI',
```

File: lib/routes/informedainews/docs.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['informedainews.com', 'informedainews.com/zh-Hans/docs/:type', 'informedainews.com/docs/:type'],
+            target: '/zh-Hans/docs/:type',
+        },
+    ],
+    name: '知闻AI',
+    maintainers: ['xirizhi'],
```
Comment (TonyRL) [inline lib/routes/informedainews/docs.ts:27]:
```suggestion
    maintainers: ['guicaiyue'],
```
This one should be your GitHub ID.

File: lib/routes/informedainews/namespace.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+export const namespace: Namespace = {
+    name: 'Informed AI News',
+    url: 'informedainews.com',
+    description: `
+:::tip
+informed AI RSS feeds:
+
+-   World News Daily: \`https://informedainews.com/docs/world-news-daily\`
+-   Tech Enthusiast Weekly: \`https://informedainews.com/docs/tech-enthusiast-daily\`
+-   AI Enthusiast Weekly: \`https://informedainews.com/docs/ai-enthusiast-daily\`
```
Comment (TonyRL) [inline lib/routes/informedainews/namespace.ts:12]: These three aren't RSS feeds.

File: lib/routes/informedainews/docs.ts [hunk 5]
```diff
@@ -0,0 +1,78 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/zh-Hans/docs/:type',
+    categories: ['journal'],
```
Comment (TonyRL) [inline lib/routes/informedainews/docs.ts:9]: This does not belong to journal.

Comment (guicaiyue) [reply lib/routes/informedainews/docs.ts]: Thank you. I'll adjust it.

**Pull request merged**